app/models/user.rb

```diff
@@ -40,6 +44,8 @@ def send_welcome_email_to_new_user
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  scope :is_admin, -> { where(role: ADMIN) }
```

Comment:
no need for this scope
remember u have enum for roles
https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html